File: AndroidStudioProject/ListExample/app/src/main/java/com/ms/listexample/MainActivity.java
package com.ms.listexample;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.AdapterView;
import android.widget.ArrayAdapter;
import android.widget.ListView;
import android.widget.Toast;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {
    ListView listView;
//    int[] arrNo = new int[]{1,2,3,4,5}; // fixed array
    ArrayList<String> arrNames = new ArrayList<>(); // not fixed(Unlimited size array)
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

       listView = findViewById(R.id.listView);

        arrNames.add("Ram");
        arrNames.add("Ramesh");
        arrNames.add("Ramu");
        arrNames.add("Ranjan");
        arrNames.add("Rajan");
        arrNames.add("Rahul");
        arrNames.add("Ram");
        arrNames.add("Ramesh");
        arrNames.add("Ramu");
        arrNames.add("Ranjan");
        arrNames.add("Rajan");
        arrNames.add("Rahul");
        arrNames.add("Ram");
        arrNames.add("Ramesh");
        arrNames.add("Ramu");
        arrNames.add("Ranjan");
        arrNames.add("Rajan");
        arrNames.add("Rahul");
        arrNames.add("Ram");
        arrNames.add("Ramesh");
        arrNames.add("Ramu");
        arrNames.add("Ranjan");
        arrNames.add("Rajan");
        arrNames.add("Rahul");

        ArrayAdapter<String> adapter = new ArrayAdapter<>(getApplicationContext(), android.R.layout.simple_list_item_1, arrNames);
        // we can also write MyActivity.this or only this on place of getApplicationContext() in above expression
        listView.setAdapter(adapter);

       listView.setOnItemClickListener(new AdapterView.OnItemClickListener() {
           @Override
           public void onItemClick(AdapterView<?> adapterView, View view, int position, long l) {
               if (position == 0)
               {
                   Toast.makeText(MainActivity.this,"Clicked on first item", Toast.LENGTH_SHORT);
               }
           }
       });

    }
}

File: KotlinProject/RecyclerView/app/src/main/java/com/ms/recyclerview/MainActivity.kt
package com.ms.recyclerview

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.recyclerview.widget.LinearLayoutManager
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        val arrContact = ArrayList<ContactModel>()
        arrContact.add(ContactModel(R.drawable.ic_launcher_background,"A","8809198023"))
        arrContact.add(ContactModel(R.drawable.ic_launcher_foreground, "B", "8790124576"))
        arrContact.add(ContactModel(R.drawable.ic_launcher_background, "C", "8956091287"))

        recyclerV.layoutManager = LinearLayoutManager(this, )
    }
}